secrets/config/hubs/schema.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: 'http://json-schema.org/draft-07/schema#'
```

Comment:
So the only things that can go here are name and config - everything else should be in the regular config file. This is because we only merge things under `config` via helm, and not the whole file. Is my understanding here correct?

Reply:
Yes, correct